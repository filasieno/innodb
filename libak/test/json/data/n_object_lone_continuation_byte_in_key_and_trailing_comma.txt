name=n_object_lone_continuation_byte_in_key_and_trailing_comma
max_depth=32
max_string_size=2048
max_json_size=1048576
----------
{"¹":"0",}