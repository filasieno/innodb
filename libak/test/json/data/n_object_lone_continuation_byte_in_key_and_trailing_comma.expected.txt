---
STATE_CHANGED_EVENT: STATE_INITIALIZED
BEGIN_OBJECT
ATTR_KEY "¹" more=0
STRING_VALUE "0" more=0
STATE_CHANGED_EVENT: STATE_ERROR 223
